tests/expectations/core/test_expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
```

Comment:
Could we put this test file under coverage in `pyproject.toml`?

Reply:
Done. Added `mypy` exception to list.